pkg/model/bootstrapscript.go

```diff
@@ -71,6 +71,17 @@ func (b *BootstrapScript) ResourceNodeUp(ig *kops.InstanceGroup) (*fi.ResourceHo
 			}
 			return ""
 		},
+
+		// Pass in configuration for the cluster proxy
+		"ProxyEnv": func() string {
+			if os.Getenv("CLUSTER_HTTP_PROXY") != "" || os.Getenv("CLUSTER_HTTPS_PROXY") != "" {
+				return fmt.Sprintf("export HTTP_PROXY=%s\nexport HTTPS_PROXY=%s\nexport NO_PROXY=%s\n",
+					os.Getenv("CLUSTER_HTTP_PROXY"),
```

Comment:
@johnzeringue here is some documentation that will help you with adding the new values into the API. https://github.com/kubernetes/kops/blob/master/docs/development/api_updates.md

I aggree with @justinsb that having these values as a new struct in the `ClusterSpec` makes a lot of sense.  Here is an example of a struct that lives in the `ClusterSpec` https://github.com/kubernetes/kops/blob/master/pkg/apis/kops/cluster.go#L225